node/src/actors/chain_manager/mining.rs

```diff
@@ -53,9 +53,9 @@ impl ChainManager {
             return;
         }
 
-        // We only want to mine in Synced state
-        if self.sm_state != StateMachine::Synced {
-            log::debug!("Not mining because node is not synced");
+        // We only want to mine in Ready state
+        if self.sm_state != StateMachine::Live {
+            log::debug!("Not mining because node is not in Ready State");
```

Comment:
Search "Ready" and replace by "Live"?

```suggestion
            log::debug!("Not mining because node is not in Live State");
```